bindata/network/ovn-kubernetes/common/ipsec-host.yaml

```diff
@@ -236,6 +236,13 @@ spec:
           ip x s flush
           ip x p flush
 
+          # Make libreswan to narrow down the proposals exchanged by two peers to a mutually acceptable set.
+          # The /etc/ipsec.d/openshift.conf file can not be used because it is managed by openvswitch.
```

Comment:
Out of curiosity, how does OVS know about `/etc/ipsec.d/openshift.conf`?

Reply:
This path is passed as `--ipsec-conf` parameter to `ovs-monitor-ipsec` script by `ovn-ipsec` container at https://github.com/pperiyasamy/cluster-network-operator/blob/master/bindata/network/ovn-kubernetes/common/ipsec-host.yaml#L257